### ISSUE TITLE
can not set language

### ISSUE THREAD

**Issue by GoogleCodeExporter** [hoanglanton/cyanogen-updater#142]
```
First this is not a Bugtracker for Cyanogen-Rom Bugs. Only for the updater.
The other Bugtracker can be found here:
http://code.google.com/p/cyanogenmod/issues/list

What steps will reproduce the problem?
1. Install cm-updater 4.5 or 4.5.1
2. Run the program
3. Interface is in polish?

What is the expected output? What do you see instead?
Inerface is in english or at least a language setting menu

What ROM are you running?(including Version)
Cyanogenmod 4.2.1
What Version of CM-Updater are you running??(When using an Experimental
one, please Provide the SVN Release Number)
4.5.1
Please provide a logcat of the issue:
If you get a Force close do an: adb logcat
If somethings wrong in the app do an: adb logcat cmupdater:V *:S
For older Versions of the updater: adb logcat | grep "<CM-Updater>"


Please provide any additional information below.

```

Original issue reported on code.google.com by `akos.sch...@gmail.com` on 29 Oct 2009 at 5:51

**Comment by GoogleCodeExporter** [comment]
```
it depends on your system setting language. what language is your phone set to?
```

Original comment by `FireFart` on 29 Oct 2009 at 6:16

**Comment by GoogleCodeExporter** [comment]
```
Issue 143 has been merged into this issue.
```

Original comment by `FireFart` on 29 Oct 2009 at 6:16

**Comment by GoogleCodeExporter** [comment]
```
My g1 is setted to Hungarian.


```

Original comment by `akos.sch...@gmail.com` on 29 Oct 2009 at 6:19

**Comment by GoogleCodeExporter** [comment]
```
that bug is fixed in 4.5.1 .. i used the wrong language code for the croation 
translations ;)
```

Original comment by `FireFart` on 29 Oct 2009 at 6:39
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `FireFart` on 23 Dec 2009 at 9:04
- Changed state: **Verified**
